.github/workflows/publish_draft_releases.yml

```diff
@@ -764,7 +1008,44 @@ jobs:
           aws-region: ${{ needs.start-node-builder-current.outputs.aws-region }}
           runner-label: ${{ needs.start-node-builder-current.outputs.runner-label }}
           aws-instance-id: ${{ needs.start-node-builder-current.outputs.aws-instance-id }}
-  start-integration-tester:
+  start-calamari-integration-tester:
+    runs-on: ubuntu-20.04
+    outputs:
+      runner-label: ${{ steps.start-self-hosted-runner.outputs.runner-label }}
+      aws-region: ${{ steps.start-self-hosted-runner.outputs.aws-region }}
+      aws-instance-id: ${{ steps.start-self-hosted-runner.outputs.aws-instance-id }}
+    if: ${{ startsWith(github.ref, 'refs/tags') || github.ref == 'refs/heads/manta' }}
```

Comment:
we'll probably want this running on branches leading to a release merge instead of the tag. i'm thinking a filter like 
`startsWith(github.ref, 'refs/heads/release-` and then we force everyone to make release branches with the `release-v1.2.3` name